README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://sich-mykhailo.github.io/layout_calendar/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/215474134-1c8e23df-0291-41f4-b963-aef30cdac91f.png)
![image](https://user-images.githubusercontent.com/38065883/215474079-388a95ef-d2fb-4731-b88c-b5c859c6e813.png)
Doesn't look like 100px square to me

Reply:
Also would be nice to add transition on hover(* task)